src/components/05_pages/NodeForm/index.js

```diff
@@ -29,6 +29,9 @@ class NodeForm extends React.Component {
       PropTypes.oneOfType([
         PropTypes.func,
         PropTypes.instanceOf(React.Component),
+
+        // Our object shape will be like PropTypes.shape({field_name: PropTypes.shape({component: PropTypes.instanceOf(React.Component), multiple: PropTypes.bool})}).
```

Comment:
Oh I'm sorry, I could have been more clear. You could have used PropTypes.shape in the actual proptype :) Does that make sense?